sdks/python/apache_beam/runners/direct/bundle_factory.py

```diff
@@ -198,3 +200,16 @@ def commit(self, synchronized_processing_time):
     self._committed = True
     self._elements = tuple(self._elements)
     self._synchronized_processing_time = synchronized_processing_time
+
+  def get_min_timestamp(self):
```

Comment:
<!--new_thread; commit:61ae92dcffa98203cbe8f56930802405de2524f0; resolved:0-->
Please match indentation of the rest of the file (2sp).